crates/iceberg-catalog/src/api/management/v1/user.rs

```diff
@@ -148,8 +148,10 @@ impl ListUsersQuery {
 }
 
 #[derive(Debug, Serialize, utoipa::ToSchema)]
+#[serde(rename_all = "kebab-case")]
 pub struct ListUsersResponse {
     pub users: Vec<User>,
+    #[serde(alias = "next_page_token")]
```

Comment:
I'm not sure if these alias annotations work for Serialize, they'd need to serialize the field twice

Reply:
Follow-up: https://github.com/lakekeeper/lakekeeper/issues/969